scripts/deploy-cloud-components.sh

```diff
@@ -39,18 +39,18 @@ faas-cli template pull
 kubectl port-forward svc/gateway -n openfaas 31111:8080 &
 sleep 2
 
-for i in {1..60};
-do
-    echo "Checking if OpenFaaS GW is up."
-
-    curl -if 127.0.0.1:31111
-    if [ $? == 0 ];
-    then
-        break
-    fi
-
-    sleep 1
-done
+# for i in {1..60};
```

Comment:
Shouldn't this stay but be replaced by the rollout / status command?

Reply:
Agree.. I have updated it.